app/models/user.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class User < ApplicationRecord
+  has_many :posts, dependent: :destroy
+  attr_accessor :remember_digest
```

Comment:
What is this doing here? Are you using it? I don't see a column in your schema for this. This value is clearly nil because it doesn't get set anywhere in your code, and this is the value you are setting your `cookies.permanent[:remember_token]` in `application_helper.rb`. Is this right?

[**REQUIRED**]